src/app/services/products.service.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import ProductInterface from '../interfaces/products';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ProductsService {
+  private baseUrl = 'assets/products.json';
+
+  constructor(private http: HttpClient) {}
+
+  getProducts(searchValue: string): Observable<ProductInterface[]> {
```

Comment:
the searchValue isn't being used inside the function or network call at all. I'm assuming this will be used once the endpoints are established?

Reply:
yes, can't really use it in a JSON file so I will add it once I have the endpoint